examples/align-two-surfaces/index.html

```diff
@@ -39,7 +39,7 @@
 
       #viewer {
         width: 100%;
-        height: 100%;
+        height: calc(100% - 250px);
```

Comment:
minor nit: could we potentially add a container to size the viewer rather than having the fixed value removed? Bit of an edge case, but if I change my Chrome font size to very large, it cuts off part of the viewer. Thinking something like:

```html
<style>
  .container {
    display: flex;
    flex-direction: column;
    width: 100%;
    height: 100%;
  }
</style>

<body>
  <div class="container>
    <!-- example setup -->
  </div>
</body>
```

Reply:
Good idea! Updated.